spec/features/create_user_spec.rb

```diff
@@ -0,0 +1,27 @@
+require "rails_helper"
+
+feature "Create user" do
```

Comment:
maybe we should make these more feature-y. So "User signs in" instead of "Create user" and "user is member of the correct GitHub team" rather than "creating a user..." (also perhaps we should rename file)